### PR TITLE
Cp7 m61 fix search parameters pagination

### PR DESCRIPTION
**Pull request by victorvassilev**

### CODE REVIEW

File: www/include/Administration/performance/viewData.php [hunk 1]
```diff
@@ -90,34 +89,27 @@
...
+        $inputs[$argumentName] = $inputGet[$argumentName];
     }
 }
 
-
-if (isset($inputs["searchH"])) {
-    $searchH = $inputs["searchH"];
-    if (isset($oreon->historySearch[$url]) && $oreon->historySearch[$url] != $searchH) {
-        $num = 0;
-        $oreon->historySearch[$url] = $searchH;
```
Comment (sc979) [inline www/include/Administration/performance/viewData.php]: why removing this method ?

File: www/include/Administration/performance/viewData.php [hunk 2]
```diff
@@ -90,34 +89,27 @@
...
+    $searchH = $_POST["searchH"];
+} elseif (isset($_GET['searchH'])) {
+    $searchH = $_GET['searchH'];
 } else {
     $searchH = null;
 }
 
-if (isset($inputs["searchS"])) {
-    $searchS = $inputs["searchS"];
-    if (isset($oreon->historySearchService[$url]) && $oreon->historySearchService[$url] != $searchS) {
```
Comment (sc979) [inline www/include/Administration/performance/viewData.php]: why removing this method ?

File: www/include/Administration/performance/viewData.php (hunk 1 above)
Comment (victorvassilev) [reply www/include/Administration/performance/viewData.php]: it was reworked, i explained in detail on chat

File: www/include/home/customViews/views.php [hunk 3]
```diff
@@ -46,6 +46,8 @@
 
 try {
 
+    require_once _CENTREON_PATH_ . "vendor/smarty/smarty/libs/Smarty.class.php";
```
Comment (kduret) [inline www/include/home/customViews/views.php]: Is it required to fix pagination ?

Comment (victorvassilev) [reply www/include/home/customViews/views.php]: no I guess it went over from the merge, I guess after including bootstrap it is not needed, so i will remove it

**Pull request merged**